libknet/threads_pmtud.c

```diff
@@ -482,7 +482,12 @@ static int _handle_check_pmtud(knet_handle_t knet_h, struct knet_host *dst_host,
 		}
 		dst_link->has_valid_mtu = 0;
 	} else {
-		dst_link->has_valid_mtu = 1;
+		if (dst_link->status.mtu < calc_min_mtu(knet_h)) {
+			log_debug(knet_h, KNET_SUB_PMTUD, "Invalid MTU detected for host: %u link: %u", dst_host->host_id, dst_link->link_id);
```

Comment:
I mean I would maybe even increase log level for that to info, as this is something worthy to see IMO - at least for our case with corosync. And maybe include the values of the min & detected MTU in the log message?